tox.ini

```diff
@@ -16,11 +16,12 @@ deps =
     statsd
 install_command = pip install {opts} {packages}
 
-[testenv:flake8]
-commands = flake8 kinto tests docs/conf.py
```

Comment:
shouldn't you keep `flake8` for py35 on Travis?

Reply:
Yes I should have removed it in travis.